packages/widgets/src/lib/components/InferenceWidget/widgets/ConversationalWidget/ConversationalWidget.svelte

```diff
@@ -125,33 +117,18 @@
 			await tick();
 			return;
 		}
-		if (!compiledTemplate) {
-			return;
-		}
 		if (!inferenceClient) {
 			error = "Inference client not ready";
 			return;
 		}
-		// Render chat template
+
 		specialTokensMap = extractSpecialTokensMap(tokenizerConfig);
 
-		let chatText;
-		try {
-			chatText = compiledTemplate.render({
-				messages,
-				add_generation_prompt: true,
-				...specialTokensMap,
-			});
-		} catch (e) {
-			error = `An error occurred while rendering the chat template: "${(e as Error).message}"`;
-			return;
-		}
+		const previousMessages = [...messages];
 
-		const input: TextGenerationInput & Required<Pick<TextGenerationInput, "parameters">> = {
-			inputs: chatText,
-			parameters: {
-				return_full_text: false,
-			},
```

Comment:
given that's recent code from @Wauplin and @SBrandeis, i would like a review from them on this